src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/action_group/test_notifications/_create.py

```diff
@@ -51,7 +51,6 @@ def _build_arguments_schema(cls, *args, **kwargs):
             required=True,
         )
         _args_schema.resource_group = AAZResourceGroupNameArg(
-            help="Name of resource group. You can configure the default group using `az configure --defaults group=<name>`.",
```

Comment:
Will this line be re-generated if running code-gen again?

Reply:
code gen has removed those redundant help message in the generated commands. [in the code](https://github.com/kairu-ms/azure-cli/blob/9ab3e155871cc0cdc83b3799d8fdc29580cf34a2/src/azure-cli-core/azure/cli/core/aaz/_arg.py#L492-L493)